jscomp/bsb/bsb_ninja_regen.ml

```diff
@@ -72,10 +72,25 @@ let regenerate_ninja
         ~name:(lib_bs_dir // Literals.sourcedirs_meta)
         config.file_groups
     ;
+#if BS_NATIVE then
+    if !Bsb_global_backend.backend = Bsb_config_types.Js then begin
+      Bsb_merlin_gen.merlin_file_gen ~per_proj_dir
+        config;       
```

Comment:
when backend = Js, do you need special handling here

Reply:
The reason we do this here is to allow bsb to fully handle JS like it does right now, and otherwise re-delegate to bsb-native